### DISCUSSION TITLE
Lab 2 Feedback

### DISCUSSION THREAD

**Discussion by rgentul** [rgentul/Interactive-Lab-Hub#5] [General]
Feedback for Lab 2 can go here. Thanks!

**Comment by rgentul** [comment]
rmk328: Very creative, really like how you used the buttons. 2 things come to mind 1) it would be cool if there was some context around adding or deleting an hour. For example, are you setting an alarm clock? Starting a cooking timer? 2) It would be better if the clock format was more readable, with just the pure string of data numbers its tough to see what hour you're in.

**Comment by rgentul** [comment]
aa653: I think you did a great job using the button functionalities and adding/subtracting time - so its very sound technically. Also the instructions were clear. I'd say a cool next step would be to not change the screen, but just have when you press the top button that "top button adds hour" becomes green text and the time changes when you release the button. So its all in one screen. Same for bottom button. That's more in line with the way alarm clocks work.

**Comment by amojena** [comment]
Your modified pi clock is what I want to do for my part G so good job haha! Maybe you could set up like a world clock thing where you have 2-3 time zones displayed on the screen?

**Comment by singhaniasnigdha** [comment]
I like the idea of using multiple button clicks for doing a bunch of things! Maybe you could build out a menu of sorts (like how we have on monitors) and use the 2 buttons to navigate around it. But then, I'm not sure if 2 buttons is enough for that :D 

Also, excited to see what images you add to your device!

Good Luck!
